.rubocop.yml

```diff
@@ -4,50 +4,113 @@ require:
   - rubocop-rspec
 
 AllCops:
-  TargetRubyVersion: 2.7
+  TargetRubyVersion: 2.7.2
```

Comment:
Do we really need such minor version?

Reply:
I do not think so. Les't try to get rid of it.